tests/ops/integration_tests/saas/test_snap_task.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+from fides.api.models.policy import Policy
+from tests.ops.integration_tests.saas.connector_runner import ConnectorRunner
+from fides.config import CONFIG
+
+
+@pytest.mark.skip(reason="Currently unable to test OAuth2 connectors")
+@pytest.mark.integration_saas
+class TestsnapConnector:
+    def test_connection(self, snap_runner: ConnectorRunner):
+        snap_runner.test_connection()
+
+    # Run erasure with masking_strict = False so both update and delete actions can be used
+    masking_strict = CONFIG.execution.masking_strict
+    CONFIG.execution.masking_strict = False
```

Comment:
This is already done for you automatically inside of the `snap_runner.non_strict_erasure_request`